test/src/tests/acceptancetests/httpInfrastructuregrouptest/httpInfrastructure_test.go

```diff
@@ -550,13 +550,13 @@ func (s *HTTPSuite) TestDelete417(c *chk.C) {
 }
 
 //429
-
+/* test is broken, server constantly returns 429, disabled until test server is fixed
```

Comment:
>oken, server constantly returns 429, disabled until test server is fixed [](start = 13, length = 72)

can you link  an opened issue in the autorest test server fro this  if there is one otheriwse can you create one please.

Reply:
By link I mean to the PR 

---
In reply to: [167278782](https://github.com/Azure/autorest.go/pull/94#discussion_r167278782) [](ancestors = 167278782)

Reply:
The PR where this was originally hit is https://github.com/Azure/autorest.go/pull/77.
The issue tracking fixing it is https://github.com/Azure/autorest/issues/2811.